dist/ExtUtils-ParseXS/lib/perlxs.pod

```diff
@@ -2208,11 +2184,949 @@ In Makefile.PL add -ltirpc and -I/usr/include/tirpc.
+
+C<switch_to_global_locale()> was written initially to cope with the
+C<Tk> library, but is general enough for other similar situations.  C<Tk>
+changes the global locale to match its expectations (later versions of
+it allow this to be turned off).  This presents a conflict with Perl
+thinking it also controls the locale.  Calling this function tells Perl to
+yield control.  Calling L<perlapi/C<sync_locale>> tells Perl to take
+control again, accepting whatever the locale has been changed to in the
+interim.  If your code is called during that interim, all portions of
+the locale will be the raw underlying values.  Should you need to
+manipulate numbers, you are on your own with regard to the radix
+character and grouping.  If you find yourself in this situation, it is
+generally best to make the interval between the calls to these two
+functions as short as possible, and avoid calculations until after perl
+has control again.
+
+It is important for perl to know about all the possible locale
+categories on the platform, even if they aren't apparently used in your
+program.  Perl knows all of the Linux ones.  If your platform has
+others, you can submit an issue at
+L<https://github.com/Perl/perl5/issues> for inclusion of it in the next
+release.  In the meantime, it is possible to edit the Perl source to
+teach it about the category, and then recompile.  Search for instances
+of, say, C<LC_PAPER> in the source, and use that as a template to add
+the omitted one.
+
+There are further complications under multi-threaded operation.  Keep on
+reading.
+
+=head2 Dealing with embedded perls and threads
+
+It is possible to embed a Perl interpreter within a larger program.  See
+L<perlembed>.
+
+MULTIPLICITY is the way this is accomplished internally; it is described in
+L<perlguts/How multiple interpreters and concurrency are supported>.
+Multiple Perl interpreters may be embedded.
+
+It is also possible to compile perl to support threading.  See
+L<perlthrtut>.  Perl's implementation of threading requires
+MULTIPLICITY, but not the other way around.
+
+MULTIPLICITY without threading means that only one thing runs at a time,
+so there are no concurrency issues, but each component or instance can
+affect the global state, potentially interfering with the execution of
+other instance.  This can happen if one instance:
+
+=over
+
+=item *
+
+changes the current working directory
+
+=item *
+
+changes the process's environment
+
+=item *
+
+changes the global locale the process is operating under
+
+=item *
+
+writes to shared memory or to a shared file
+
+=item *
+
+uses a shared file descriptor (including a database iterator)
+
+=item *
+
+raises a signal that functions in other instances are sensitive to
+
+=back
+
+If your code doesn't do any of these things, nor depends on any of their
+values, then Congratulations!!, you don't have to worry about MULTIPLICITY
+nor threading.  But wait, a surprising number of libc functions do
```

Comment:
"nor" without "neither" is rare, but it is valid. See method 2 at https://www.wikihow.com/Use-Nor

If it were to be rephrased to avoid "nor" I think it would need to be:
```
If your code doesn't do any of these things and does not depend on any of their
values, then Congratulations!!, you don't have to worry about MULTIPLICITY
nor threading.
```

Reply:
Well, that still retains a 'nor'.  Shouldn't the last phrase be:
```
you don't have to worry about MULTIPLICITY
or threading.
```

Reply:
I still like the original.  Being a dinosaur, I consulted Strunk and White "The Elements of Style" and Fowler's "Modern English Usage", and neither is not required for nor.  I prefer nor here because it emphasizes the negative aspect.

Reply:
changed